source/includes/_skydb.md

```diff
@@ -1,114 +1,85 @@
 # SkyDB
 
 SkyDB is a framework that allows Skynet users to create decentralized accounts
-and store mutable files in those accounts.
-
-<aside class="warning">
-These functions are in beta and may change in the coming weeks.
-</aside>
+and store mutable files in those accounts. It is built on top of the [Registry](#registry).
```

Comment:
I would mention that every entry in registry for skydb is a skylink that contains the data.